src/app/pages/home/connect/connect.component.html

```diff
@@ -1,34 +1,31 @@
+<ion-header>
+    <ion-toolbar color="primary">
+        <ion-buttons slot="start">
+            <ion-menu-button menu="m1"></ion-menu-button>
+        </ion-buttons>
+        <ion-title>Connect</ion-title>
+    </ion-toolbar>
+</ion-header>
 <ion-content>
-    <div class="banner">
-        this is the banner
-    </div>
-    <!-- <div class="connect-groups">
-        <ion-grid>
-            <ion-row *ngFor="let group of connectGroups">
-                <ion-col size="3">
-                    {{ group.name }}
-                </ion-col>
-                <ion-col>
-                    {{ group.description }}
-                </ion-col>
-            </ion-row>
-        </ion-grid>
-    </div> -->
-
-    <ion-list class="connect-groups">
+    <ion-list>
         <ion-item *ngFor="let group of connectGroups">
             <ion-grid>
+                <br/>
                 <ion-row>
-                    <ion-col size="3">
-                        {{ group.name }}
+                    <ion-col size="10">
+                        <ion-row>
+                            {{ group.name }}
+                            <br/>
+                            {{ group.description }}
+                        </ion-row>
                     </ion-col>
                     <ion-col>
-                        {{ group.description }}
+                        <br/>
+                        <ion-icon name="arrow-dropright"></ion-icon>
```

Comment:
Can this be a self-closing tag?

Reply:
the br or ion-icon?

Reply:
br definitely

Reply:
the `ion-icon`, i think its a HTML standard where all tags can be self-closing